.gitignore

```diff
@@ -1 +1,4 @@
 gin-swagger
+
+.idea
+vendor
```

Comment:
`vendor` is intentionally checked into this repository to allow `go get` of the `gin-swagger` binary.

Reply:
This is something I don't understand to be honest. Was thinking that a `go get https://github.com/mikkeloscar/gin-swagger` would be possible without having `vendor` folder checked in.